website/content/docs/enterprise/ent-to-ce-downgrades.mdx

```diff
@@ -0,0 +1,144 @@
+---
+layout: docs
+page_title: Downgrade from Consul Enterprise to the community edition
+description: >-
+  Learn how to downgrade your installation of Consul Enterprise to the free Consul community edition (CE)
+---
+
+# Downgrade from Consul Enterprise to the community edition
+
+This document describes how to downgrade from Consul Enterprise to Consul community edition (CE).
+
+## Download the CE binary version
+
+First, download the binary for the CE version you want.
+
+<Tabs>
+<Tab heading="Binary">
+
+All current and past versions of the CE and Enterprise releases are
+available on the [HashiCorp releases page](https://releases.hashicorp.com/consul)
+
+</Tab>
+<Tab heading="Docker">
+
+Docker containers are available on [Docker Hub](https://hub.docker.com/r/hashicorp/consul)
+
+</Tab>
+
+</Tabs>
+
+## Prepare for the downgrade to CE
+
+1. Take a snapshot:
+1. Inspect the snapshot to ensure that Consul successfully captured it using the following command:
+this snapshot somewhere safe. Refer to the following documentation for additional information about creating and using snapshots:
+
+    - [Consul Snapshot](/consul/commands/snapshot)
+    - [Backup Consul Data and State tutorial](/consul/tutorials/production-deploy/backup-and-restore)
+
+1. Temporarily modify your Consul configuration so that its [log_level](/consul/docs/agent/config/cli-flags#_log_level)
+is set to `debug`. After doing this, issue the following command on your servers to
+reload the configuration:
+1. Remove references to services in non-default namespaces and partitions from service resolver, service splitter, and service router configuration entries.
```

Comment:
I think this step would need more details about what must be done. Some of the immediate questions I have are:

* Should the config-entries in non-default namespaces/partitions be deleted?
* Should the non-default namespaces be deleted fully? 
* Or does this only apply to config entries in the default namespace and partition?

Reply:
the target services(of redirects,failovers,splits,routes) belonging to non default tenancies should be removed

Reply:
should i specify which parts of the config entry you need to remove (like failovers and redirects for service resolver) for non default tenancies ?

Reply:
have detailed the individual entries more.